job-controller/src/main/java/feast/jobcontroller/runner/ConsolidatedJobStrategy.java

```diff
@@ -71,6 +74,9 @@ private String createJobId(SourceProto.Source source) {
                 source.getKafkaSourceConfig().getBootstrapServers(),
                 source.getKafkaSourceConfig().getTopic()),
             dateSuffix);
+    if (!this.jobProperties.getJobIdPrefix().isEmpty()) {
```

Comment:
What happens when jobProperties is null?

Reply:
Addressed case.